packages/react-examples/src/react-popup/Popup/Popup.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { Popup, PopupTrigger, PopupContent, PopupProps } from '@fluentui/react-popup';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  example: {
+    padding: '12px',
+  },
+});
+
+const ExampleContent = () => {
+  const { example } = useStyles();
+  return (
+    <div className={example}>
+      <h3>Popup content</h3>
+
+      <div>This is some popup content</div>
+    </div>
+  );
+};
+
+export const Default = (props: PopupProps) => (
+  <Popup {...props}>
+    <PopupTrigger>
+      <button>Popup trigger</button>
+    </PopupTrigger>
+
+    <PopupContent>
+      <ExampleContent />
+    </PopupContent>
+  </Popup>
+);
+
+Default.argTypes = {
+  openOnContext: {
+    defaultValue: false,
+    control: 'boolean',
+  },
+
+  openOnHover: {
+    defaultValue: false,
+    control: 'boolean',
+  },
+};
+
+export const AnchorToTarget = () => {
+  const [target, setTarget] = React.useState<HTMLButtonElement | null>();
+
+  return (
+    <>
+      <div>
+        <Popup target={target}>
+          <PopupTrigger>
+            <button>Popup trigger</button>
+          </PopupTrigger>
+
+          <PopupContent>
+            <ExampleContent />
+          </PopupContent>
+        </Popup>
+      </div>
+
+      <button style={{ marginTop: 100 }} ref={setTarget}>
+        Custom target
+      </button>
+    </>
+  );
+};
+
+export const Controlled = () => {
+  const [open, setOpen] = React.useState(false);
+  const onOpenChange: PopupProps['onOpenChange'] = (_, data) => setOpen(data.open || false);
+
+  return (
+    <>
+      <Popup open={open} onOpenChange={onOpenChange}>
+        <PopupTrigger>
+          <button>Custom trigger</button>
+        </PopupTrigger>
+        <PopupContent>
+          <ExampleContent />
+        </PopupContent>
+      </Popup>
+    </>
+  );
+};
+
+export const WithCustomTrigger = () => {
+  const [open, setOpen] = React.useState(false);
+  const [target, setTarget] = React.useState<HTMLElement | null>(null);
+  const onClick = () => setOpen(s => !s);
+  const onKeyDown = (e: React.KeyboardEvent) => {
```

Comment:
Can we use a `button` as a custom trigger instead? Stories are considered as our docs now and I don't think that we should spread such examples to wide audience 😉

Reply:
that's fair, the intention was to show that `PopupTrigger` is not required for this component to work